cmd/mesh/manifest.go

```diff
@@ -89,6 +91,19 @@ func genManifests(inFilename string, setOverlayYAML string) (name.ManifestMap, e
 	if err != nil {
 		return nil, err
 	}
+	if util.IsHTTPURL(mergedICPS.InstallPackagePath) {
+		// TODO: Make the filename logic more robust
+		uf, err := helm.NewURLFetcher(mergedICPS.InstallPackagePath, "", helm.InstallationChartsFileName, helm.InstallationShaFileName)
+		if err != nil {
+			return nil, err
+		}
+		errs := uf.FetchBundles()
+		if len(errs) != 0 {
+			return nil, errs.ToError()
+		}
+		mergedICPS.InstallPackagePath = filepath.Join(uf.GetDestDir(), "install", "kubernetes", "operator", "charts")
```

Comment:
you don't have to separate the path elements for filepath i.e. just "install/kubernetes/operator/charts" - but that path should come from a const, which you can put in here for now.

Reply:
done